pkg/scheduler/framework/interface.go

```diff
@@ -74,6 +74,15 @@ const (
 	Skip
 )
 
+// statusPrecedence defines a map from status to its precedence, larger value means higher precedent.
```

Comment:
sorry, last nit: have this after `var codes` (because it references the enum).

Reply:
Good catch, done